cluster/images/hyperkube/static-pods/master-multi.json

```diff
@@ -12,13 +12,15 @@
       "name": "controller-manager",
       "image": "REGISTRY/hyperkube-ARCH:VERSION",
       "command": [
-              "/hyperkube",
-              "controller-manager",
-              "--master=127.0.0.1:8080",
-              "--service-account-private-key-file=/srv/kubernetes/server.key",
-              "--root-ca-file=/srv/kubernetes/ca.crt",
-              "--min-resync-period=3m",
-              "--v=2"
+        "/hyperkube",
+        "controller-manager",
+        "--master=127.0.0.1:8080",
```

Comment:
kubeconfig are mostly for the workers, but it might make sense to use it everywhere.
Also, I think we could expose the certs to the host via hostPath, don't you think?